[Modules] Format the metadata.json and aligned the Name and Description for all Modules and Updated Readme (#3309)
Azure/ResourceModules: modules/network/network-managers/security-admin-configurations/README.md

> # AVNM Security Admin Configurations `[Microsoft.Network/networkManagers/securityAdminConfigurations]`

Reviewer: ```suggestion
# Network Manager Security Admin Configurations `[Microsoft.Network/networkManagers/securityAdminConfigurations]`
```

> This module deploys an Azure Virtual Network Manager (AVNM) Security Admin Configuration.

Reviewer: ```suggestion
This module deploys a Network Manager Security Admin Configuration.
```
